lms/templates/learner_dashboard/program_details_sidebar.underscore

```diff
@@ -0,0 +1,10 @@
+<aside class="aside js-program-progress program-progress">
+    <% if (programCertificate) { %>
+        <h2 class="progress-heading certificate-heading"><%- interpolate(gettext('Your %(program)s Certificate'), {program: type}, true) %></h2>
```

Comment:
Nit: this should ideally use ``StringUtils.interpolate`` as we're hoping to remove the unnamespaced version:

http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/conventions/safe_templates.html#javascript-files